openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -29,7 +29,9 @@ body {
 
 ol,
 ul {
-  padding-inline-start: 20px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
@ShaileshParmar11 can you check if ol is rendering properly or not?

Reply:
updated the screenshort